src/workerd/util/sqlite.c++

```diff
@@ -9,28 +9,29 @@
 #include <sys/stat.h>
 #include <sqlite3.h>
 #include <kj/vector.h>
+#include <workerd/jsg/jsg.h>
 
 namespace workerd {
 
 #define SQLITE_CALL_NODB(code, ...) do { \
     int _ec = code; \
-    KJ_ASSERT(_ec == SQLITE_OK, "SQLite failed: " #code, sqlite3_errstr(_ec), ##__VA_ARGS__); \
+    JSG_ASSERT(_ec == SQLITE_OK, Error, "SQLite failed: " #code, " ", sqlite3_errstr(_ec), ##__VA_ARGS__); \
   } while (false)
 // Make a SQLite call and check the returned error code. Use this version when the call is not
 // associated with an open DB connection.
 
 #define SQLITE_CALL(code, ...) do { \
     int _ec = code; \
     /* SQLITE_MISUSE doesn't put error info on the database object, so check it separately */ \
-    KJ_ASSERT(_ec != SQLITE_MISUSE, "SQLite misused: " #code, ##__VA_ARGS__); \
-    KJ_ASSERT(_ec == SQLITE_OK, "SQLite failed: " #code, sqlite3_errmsg(db), ##__VA_ARGS__); \
+    JSG_ASSERT(_ec != SQLITE_MISUSE, Error, "SQLite misused: " #code, ##__VA_ARGS__); \
```

Comment:
I think that MISUSE errors should remain as internal errors. These errors generally flag situations that are otherwise UB, so we need to make sure there's no way to cause them from JavaScript code.